modules/publicize/publicize-jetpack.php

```diff
@@ -464,7 +464,7 @@ function save_publicized( $post_ID, $post, $update ) {
 				 *
 				 * @param int The post ID
 				 */
-				do_action( 'jetpack_publicize_post', $post->ID );
+				do_action_deprecated( 'jetpack_publicize_post', $post->ID, '4.8.0', 'jetpack_published_post_flags' );
```

Comment:
For anyone else reviewing this, `do_action_deprecated()` [merged in to Core at 4.6.0](https://github.com/WordPress/WordPress/blob/dd6da701b286579819cd6aa518aa2d7018efd759/wp-includes/plugin.php#L605) and as we support current and current - 1 (currently 4.7 and 4.6) we're fine to use this. 👍